hw-latest/fpga/io_module/io_module.c

```diff
@@ -0,0 +1,64 @@
+// Licensed under the Apache-2.0 license
```

Comment:
Are you sure these drivers are necessary? It seems like I can access this memory via /dev/mem from userspace.

Reply:
You _can_ access the memory through /dev/mem, but is it a good idea? I thought that UIO would be preferred.
I could see changing the rom backdoor to be part of the UIO driver, which would simplify setup.

Reply:
If we don't need interrupts, I think it's preferable to avoid dependencies on kernel modules, since that complicates deployment.

This is fine for now, however.

Reply:
Added to my TODO list. Marking as resolved to move on with merge.